chapter2/tests/t19/terraform/base/config.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">= 1.5.0"
+
+  required_providers {
+    docker = {
```

Comment:
Not sure, but Ithink `docker` provider is redundant here

Reply:
> Not sure, but Ithink `docker` provider is redundant here

@kseniyashaydurova 
Yes, done